packages/core/src/Lib/UseColor/Internal/Lib/UseTextColor.ts

```
@@ -1,16 +1,20 @@
-import type { Color, ColorableTextComponentName, FlyonUIVueAppDefaultConfig } from '@/Lib';
-import type { ComputedRef, MaybeRefOrGetter, Ref }                            from 'vue';
-import { useColor }                                                           from '@/Lib/UseColor/Internal';
-import { isDefined }                                                          from '@/Lib/Utils/Internal';
-import { computed, toValue }                                                  from 'vue';
+import type { FlyonUIVueAppDefaultConfig, TextColor, TextColorableComponentName } from '@/Lib';
+import type { ComputedRef, MaybeRefOrGetter, Ref }                                from 'vue';
```

ESLint

[error] Expected "vue" (type) to come before "@/Lib" (internal-type).

(perfectionist/sort-imports)